packages/@aws-cdk/aws-cognito/lib/user-pool-client.ts

```diff
@@ -297,4 +306,8 @@ export class UserPoolClient extends Resource implements IUserPoolClient {
     }
     return undefined;
   }
+
+  private configurePreventUserExistenceErrors(prevent?: boolean): string | undefined {
+    return prevent === undefined || prevent ? 'ENABLED' : 'LEGACY';
```

Comment:
Return `undefined` when `props.preventUserExistenceErrors` is `undefined`. 
This way, there are no changes to any existing stacks.